logging/logging.go

```diff
@@ -6,20 +6,19 @@ import (
 	"os"
 
 	hclog "github.com/hashicorp/go-hclog"
+	"github.com/probr/probr-sdk/config"
 )
 
 var (
 	activeLogger hclog.Logger
-	logger       map[string]hclog.Logger
+	loggers      map[string]hclog.Logger
 )
 
 func init() {
 	// Initialize default logger
-	name := ""
-	logger = make(map[string]hclog.Logger)
-	logger[name] = newHCLogger(io.Writer(os.Stderr), false)
-	writer := logger[name].StandardWriter(&hclog.StandardLoggerOptions{InferLevels: true})
-	SetLogWriter(name, writer)
+	name := "default"
+	loggers = make(map[string]hclog.Logger)
+	UseLogger(name)
```

Comment:
This summarizes the bulk of the changes: just run `UseLogger(args)` and you'll create/get a logger _and_ set it as the primary logger for `log.Print()` and so forth.